### PR TITLE
Bugfix/fpm ldap library ext

### PR DESCRIPTION
**Pull request by tobybatch**
The LDAP test just loads the front page.  This will fail if the LDAP ext isn't present.

LDAP is _very_ widely used, I helped Kevin P with the initial inclusion in the kimai base project.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -108,11 +109,13 @@ RUN docker-php-ext-install -j$(nproc) zip
 FROM php:7.3.10-fpm-alpine3.10 AS fpm-alpine-base
 RUN apk add --no-cache \
         bash \
+        freetype \
         haveged \
         icu \
+        libldap \
```
Comment (Schrolli91) [inline Dockerfile:115]: sounds ok for me

File: Dockerfile [hunk 2]
```diff
@@ -108,11 +109,13 @@ RUN docker-php-ext-install -j$(nproc) zip
...
         libpng \
         libzip \
-        freetype && \
+        openldap-dev && \
```
Comment (Schrolli91) [inline Dockerfile]: is `libldap` AND `openldap` needed? (duplicate?)
why the `dev` lib in the alpine base build? is there a `openldap`-only available? (should be much smaller in footprint)

**Review by Schrolli91 — CHANGES_REQUESTED** [review]
I'm a little bit confused about the LDAP libs that we are using. Especially at which places they are installed.

File: Dockerfile (hunk 2 above)
Comment (tobybatch) [reply Dockerfile]: I was able to remove the openldap package completely.

File: Dockerfile [hunk 3]
```diff
@@ -114,8 +114,7 @@ RUN apk add --no-cache \
         icu \
         libldap \
         libpng \
-        libzip \
-        openldap && \
```
Comment (Schrolli91) [inline Dockerfile]: so if im right, we only need the libldap?
in the php ext base build for alpine is still an openldap - also not needed?

**Review by Schrolli91 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (tobybatch) [reply Dockerfile]: I removed that too, you were right.

Comment (Schrolli91) [reply Dockerfile]: forgotten to check-in this changes?

**Comment by tobybatch** [comment]
@Schrolli91 This still doesn't work

**Pull request closed without merging**